integrations/huggingface-transformers/tutorials/text-classification/text-classification-cli.md

```diff
@@ -0,0 +1,583 @@
+recipe_path = zoo_model.recipes.default.path
+print(recipe_path)
+```
+
+### **Export to ONNX**
+
+Once you have trained your model, export to ONNX in order to deploy with DeepSparse. The artifacts of the training process 
+are saved to your local filesystem. 
+
+Run the following to convert your PyTorch checkpoint to ONNX:
+
+```bash
+sparseml.transformers.export_onnx \
+  --model_path sparse_quantized_bert-text_classification_mnli \
+  --task text_classification
+```
+
+A `deployment` folder is created in your local directory, which has all of the files needed for deployment with DeepSparse including the `model.onnx`, `config.json`, and `tokenizer.json` files.
+
+## Sparse Transfer Learning onto QQP (GLUE Task)
+
+To train with the `QQP` dataset instead of `MNLI`, we can swap `--task_name mnli` for 
+`--task_name qqp` with the appropriate `recipe` and `distill_teacher`
+
+### Run Transfer Learning
+
+We will fine-tune a [90% pruned version of BERT](zoo:nlp/masked_language_modeling/obert-base/pytorch/huggingface/wikipedia_bookcorpus/pruned90-none) onto QQP.
+
+Run the following:
+```bash
+sparseml.transformers.train.text_classification \
+  --task_name qqp \
+  --model_name_or_path zoo:nlp/masked_language_modeling/obert-base/pytorch/huggingface/wikipedia_bookcorpus/pruned90-none \
+  --recipe zoo:nlp/text_classification/obert-base/pytorch/huggingface/qqp/pruned90_quant-none \
+  --distill_teacher zoo:nlp/text_classification/obert-base/pytorch/huggingface/qqp/base-none \
+  --output_dir obert_base_pruned90_quant_qqp \
+  --do_train --do_eval --evaluation_strategy epoch --logging_steps 1000 --save_steps 1000 \
+  --per_device_train_batch_size 32 --per_device_eval_batch_size 32 --preprocessing_num_workers 32 \
+  --max_seq_length 128 \
+  --seed 10194
+```
+
+Let's discuss the key arguments:
+- `--task_name qqp` instructs SparseML to download and fine-tune onto the QQP dataset. You can pass any GLUE task to this parameter.
+
+- `--model_name_or_path zoo:nlp/masked_language_modeling/obert-base/pytorch/huggingface/wikipedia_bookcorpus/pruned90-none` specifies the starting checkpoint for the fine tuning. Here, we passed a SparseZoo stub identifying the 90% pruned version of BERT trained with masked language modeling, which SparseML downloads when the script starts.
+
+- `--recipe zoo:nlp/text_classification/obert-base/pytorch/huggingface/qqp/pruned90_quant-none` specifies the recipe to be applied by SparseML. Here, we passed a SparseZoo stub identifying the transfer learning recipe for the QQP dataset, which SparseML downloads when the script starts. See below for the details of what this recipe looks like.
+
+- `--distill_teacher zoo:nlp/text_classification/obert-base/pytorch/huggingface/qqp/base-none` is an optional argument that specifies a model to use for as a teacher to apply distillation during the training process. We passed a SparseZoo stub identifying a dense BERT model trained on QQP, which SparseML downloads when the script starts.
```

Comment:
an optional argument that specifies a model to use ~~for~~ as a teacher for applying distillation during training.